ddtrace/tracer/option.go

```diff
@@ -230,7 +230,7 @@ func newConfig(opts ...StartOption) *config {
 		c.httpClient = udsClient(c.agentURL.Path)
 		c.agentURL = &url.URL{
 			Scheme: "http",
-			Host:   fmt.Sprintf("UDS_%s", strings.ReplaceAll(c.agentURL.Path, "/", "_")),
+			Host:   fmt.Sprintf("UDS_%s", strings.NewReplacer(":", "_", "/", "_", `\`, "_").Replace(c.agentURL.Path)),
```

Comment:
any reason to switch from `ReplaceAll` to `NewReplacer`?

Reply:
Yes, I want to replace multiple characters (`:`, `/`, and `\`) with underscores, anywhere in the string. `ReplaceAll` replaces occurrences of a single substring with another string.

Reply:
Oooooooohhhh I missed that 👍